src/group/index.js

```diff
@@ -30,6 +55,16 @@ exports = {
   },
 
   /**
+   * Automatically obtain a group for this class, and return an actor instance from that group.
+   * This function provides an easy way to add custom actors to scene.
+   * @func scene.addCustomActor
+   * @arg     {Actor}  ctor
+   * @arg     {object} [opts]
+   * @returns {Actor}  newInstance
+   */
+  addCustomActor: addCustomActor,
```

Comment:
Not sure - we should decide what the contribution guidelines are for scene...  longer discussion required